### PR TITLE
Add Load functions for LinkToken and VRFConsumerV2

### PR DESCRIPTION
**Pull request by tateexon**
Also some cleanup to contain all the info around proving keys and hashes for JobInfo

### CODE REVIEW

File: contracts/ethereum_contracts.go [hunk 1]
```diff
@@ -570,6 +570,18 @@ func (l *EthereumLinkToken) TransferAndCall(to string, amount *big.Int, data []b
 	return tx, l.client.ProcessTransaction(tx)
 }
 
+// LoadWithAddress load an EthereumLinkToken with a specific address
```
Comment (kalverra) [inline contracts/ethereum_contracts.go:573]: Plurality

LoadWithAddress **load** an EthereumLinkToken

File: contracts/ethereum_contracts.go [hunk 2]
```diff
@@ -1179,6 +1191,19 @@ func (v *EthereumVRFConsumerV2) GetAllRandomWords(ctx context.Context, num int)
 	return words, nil
 }
 
+// LoadWithAddress loads an EthereumVRFConsumerV2 with a specified address
+func (v *EthereumVRFConsumerV2) LoadWithAddress(address string, client blockchain.EVMClient) error {
+	a := common.HexToAddress(address)
+	consumer, err := ethereum.NewVRFConsumerV2(a, client.(*blockchain.EthereumClient).Client)
```
[the remark this answers is not in the document]
Comment (tateexon) [reply contracts/ethereum_contracts.go:1197]: The whole point that I need to pass in the specific address in. v.address is empty when I am calling LoadWithAddress.

[the remark this answers is not in the document]
Comment (tateexon) [reply contracts/ethereum_contracts.go:1197]: Example usage: https://github.com/smartcontractkit/atlas/blob/qa_soak_vrfv2_2/e2e_tests/atlas/vrfv2soak/vrfv2_soak_alerts_test.go#L106

[the remark this answers is not in the document]
Comment (tateexon) [reply contracts/ethereum_contracts.go:1197]: Example usage:
`consumer := &contracts.EthereumVRFConsumerV2{}`
`consumerAddress := os.Getenv("CONSUMER_ADDRESS")`
`consumer.LoadExistingConsumer(consumerAddress, evmClient)`

**Review by kalverra — APPROVED** [review]

**Review by skudasov — APPROVED** [review]

**Pull request merged**